README.md

```diff
@@ -125,6 +125,22 @@ The loglevel API is extremely minimal. All methods are available on the root log
 
   These enable or disable all log messages, and are equivalent to log.setLevel("trace") and log.setLevel("silent") respectively.
 
+* A `log.getLevel()` method.
+
+  Returns the current logging level.
+
+  For example, if you do string concatenation in your logging statements that
+  are in sections of code that get called a lot, Javascript does the
+  relatively-expensive string concatenation even if the current logging level
+  won't actually output the string. So here is how to prevent the unnecessary
+  overhead through a condition on the current log level:
+
+  ```javascript
+  if (log.getLevel() >= log.levels.DEBUG) {
+    log.debug("Some " + "string " + "concatenation " + "called " + "a " + "lot");
+  }
```

Comment:
Can we come up with a better example here, with an actually expensive operation? This one isn't too great, because:

A) String concatenation is pretty cheap, and I'd be surprised if string concatenation in your logging is seriously the performance bottleneck in anybody's application. (Although if that is the case for you, I'd love to see the profiling data)
B) Even if this is the bottleneck, you can instead do `log.debug("Some ", "string ", "concatenation ", "called ", "a ", "lot")` (commas instead of pluses), and the console API will do concatenation for you, and automatically do this optimization too. It'll only concatenate the strings if debug logging is enabled, as if the level is higher than debug then log.debug is a no-op, so all the args are just thrown away.

I think this example needs to log something expensive, and the explanation shouldn't be about string concatenation either: it should be more general, and we should explicitly point out that most people won't need this, and you should only use it if they have measurements showing that logging is causing performance problems. I'm keen to avoid encouraging people into bad performance practices!